util/win32-time.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2013 Martin Willi
+ * Copyright (C) 2013 revosec AG
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.  See <http://www.fsf.org/copyleft/gpl.txt>.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * for more details.
+ */
+
+#include "win32-time.h"
+
+/**
+ * timersub(3) from <sys/time.h>
+ */
+void timersub(struct timeval *a, struct timeval *b,

```

Comment:
Style: In function implementations, place return type and function name in separate lines. If the argument list exceeds 80 columns (which is the case here I think), put arguments in separate lines as well:

```c
void
timersub(
  const struct timeval *a,
  const struct timeval *b,
  struct timeval *res)
{

```